### PR TITLE
Implement sorting for Endpoints index

### PR DESCRIPTION
**Pull request by bastilian**
This implements sorting for the endpoint index for the columns, "Name", "Path (url)" and "Active" for now. 

Screen Shot 2019-04-09 at 14 54 16

### CODE REVIEW

File: src/store/reducers/reducerHelper.js [hunk 1]
```diff
@@ -86,10 +86,37 @@ export const includeRelationships = (normalizedPayload) => {
...
+            comparison = -1;
+        }
+
+        return order === 'desc' ? comparison * -1 : comparison;
+    };
+}
+
+export const normalizeData = (data, property, endpoint, sortBy) => {
+    let normalizedData = normalizePayload(data, endpoint)[property];
+    normalizedData = _.mapValues(normalizedData, (item) => { return Object.assign(item, item.attributes); });
```
Comment (adamruzicka) [inline src/store/reducers/reducerHelper.js]: Wouldn't `(item) => Object.assign(item, item.attributes)` work here?

**Review by adamruzicka — COMMENTED** [review]
One nitpick, otherwise looks good and seems to work fine.

File: src/store/reducers/reducerHelper.js (hunk 1 above)
Comment (bastilian) [reply src/store/reducers/reducerHelper.js]: We can do even better, with `(item) => ({ ...item, ...item.attributes })` :)

**Comment by adamruzicka** [comment]
Thanks @bastilian !

**Pull request merged**